client/src/Components/Media/Media.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Media } from "reactstrap";
+
+import usersData from "../../Data/profileData.json";
+
+var notesList = usersData.notes;
+
+import "./Media.css";
```

Comment:
Importing the CSS here will raise an ugly error referred as: `"Import in body of module; reorder to top import/first” `

Fixing this is simple: Just declare the `notesList` variable after you import the `Media.css` file and the build error would be mitigated.